### DISCUSSION TITLE
Login problem

### DISCUSSION THREAD

**Discussion by thamizhthenvalavan** [drkNsubuga/PharmaSpot#41] [Q&A] [answered]
Pharmaspot executable I am facing a problem with default username and password

**Comment by drkNsubuga** [comment, marked as answer]
Hi!
Kindly refer to the response to issue #40

**Comment by thamizhthenvalavan** [comment]
OS windows 8.1 and Pharmaspot versions V.1.4.3, V.1.4.2 and V.1.4.1. I have
tried all these versions with the default username: admin
password: admin

I always get an error message Invalid username or password
